Formula/darkice.rb

```diff
@@ -21,6 +22,9 @@ class Darkice < Formula
   depends_on "jack"
 
   def install
+    ["aacPlusEncoder.cpp", "FaacEncoder.cpp", "OpusLibEncoder.cpp", "VorbisLibEncoder.cpp"].each do |f|
```

Comment:
Thanks for asking. I was going to say "no", but then looked around and found [this PR](https://github.com/rafael2k/darkice/pull/120). So yes, this has been reported